README.md

```diff
@@ -15,16 +15,43 @@ This tool generate tree graph from ARF xml report from OpenSCAP scan.
   * [inquirer](https://pypi.org/project/inquirer/)
 
 ### Installation
+#### Fedora 30 and later
+```bash
+dnf install oval-graph
+# Install inquirer, if you want have nice cli futures. (optional)
+pip3 install inquirer
+```
+#### Install via [pip](https://pypi.org/project/oval-graph/)
+```bash
+pip3 install --user oval-graph
+# Install inquirer, if you want have nice cli futures. (optional)
+pip3 install --user inquirer
 
+# If you want to install oval-graph globally instead, you have to run the above command as admin, e.g. on Linux:
+sudo pip3 install oval-graph
+# Install inquirer, if you want have nice cli futures. (optional)
+sudo pip3 install inquirer
+
+```
+#### Install from source
 ```bash
 git clone https://github.com/OpenSCAP/OVAL-visualization-as-graph.git
 cd OVAL-visualization-as-graph
 
-# Install package with nice cli futures (recommended)
+# Install with nice cli futures (recommended)
+pip3 install --user ".[niceCli]"
```

Comment:
I wouldn't repeat those commands two times - once with `--user` and once without. Let's just show it once, or mention that for normal user install, "use `install --user` instead of jus `install`".